src/cuda_kernels/ConstantsG.cuh

```diff
@@ -0,0 +1,69 @@
+/**
+ * This is simply taken from the cpptraj implementation by Daniel R. Roe and Mark J. Williamson.
```

Comment:
@jokr91 Why not just refer to Constants.h in the `src` directory?

Reply:
Sorry, I removed this file.